pkg/rpcservice/sample_create_test.go

```diff
@@ -27,7 +27,13 @@ func TestSampleCreateSucceeds(t *testing.T) {
 		}, nil
 	}
 
-	createSample = func(config *config.Config, sampleName string, selectedConfig *samples.SelectedConfig, destination string, forceRefresh bool, resultChan chan<- samples.CreationResult) {
+	createSample = func(ctx context.Context,
+		config *config.Config,
+		sampleName string,
+		selectedConfig *samples.SelectedConfig,
+		destination string,
+		forceRefresh bool,
+		resultChan chan<- samples.CreationResult) {
```

Comment:
Nit: go style typically prefers all arguments on one line or all arguments on their own line as below:

```suggestion
	createSample = func(
		ctx context.Context,
		config *config.Config,
		sampleName string,
		selectedConfig *samples.SelectedConfig,
		destination string,
		forceRefresh bool,
		resultChan chan<- samples.CreationResult,
	) {
```

Reply:
Gotcha, updated!